infrastructure/quick-deploy/aws/armonik/parameters.tfvars

```diff
@@ -209,9 +209,14 @@ authentication = {
   require_authorization   = false
 }
 
+object_storage_adapter = "Redis"
+table_storage_adapter  = "ArmoniK.Adapters.MongoDB.TableStorage"
+queue_storage_adapter  = "ArmoniK.Adapters.Amqp.QueueStorage"
+
```

Comment:
Can we standardize so that we just put the name of the desired service (object_storage_adapter = "Redis", table_storage_adapter = "MongoDB", queue_storage_adapter = "Amqp") and calculate the interface/adapter in armonik module?